src/System.Text.Json/src/System/Text/Json/Serialization/JsonCamelCaseNamePolicy.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    internal sealed class JsonCamelCaseNamePolicy : JsonNamingPolicy
+    {
+        public override string ConvertName(string name)
+        {
+            if (string.IsNullOrEmpty(name) || !char.IsUpper(name[0]))
+            {
+                return name;
+            }
+
+            char[] chars = name.ToCharArray();
+
+            for (int i = 0; i < chars.Length; i++)
```

Comment:
Why not start with i = 1?

Reply:
There is a special case for 1 below

Reply:
It would require an extra `ToLower(chars[0])` which doesn't seem worth it IMO.